deploy/examples/monitoring/prometheusrule-default-values.yaml

```
@@ -0,0 +1,117 @@
+CephMgrIsAbsent:
+  for: 5m
+  namespace: rook-Ceph
```

The default exaamples use `rook-ceph` for the default namespace, but really the operator should fill in this value dynamically so it doesn't have to be filled in statically here. Can the operator post-process to fill in the namespace?
```suggestion
  namespace: rook-ceph
```

---

> Can this wait until v1.9 (targeted for end of March)? This is a potentially impactful change and downstream dependents may need time to adjust to the customizations. It might also give us time to pick up the new alert rules coming soon. If v1.9 is ok, please also add an entry to PendingReleaseNotes.md. Thanks!
> 
> edit: to be clear, changes for v1.9 can be merged whenever, just not backported to release branches.

Sure, we can wait until v1.9. I'll add an entry to PendingReleaseNotes.md.